public/app/features/correlations/utils.test.ts

```diff
@@ -68,10 +70,27 @@ describe('correlations utils', () => {
     // Prometheus value (linked to Elastic)
     expect(testDataFrames[2].fields[0].config.links).toHaveLength(1);
   });
+
+  it('changes the config field if loki dataplane is being used and the correlation is pointing to the legacy body field (Line)', () => {
+    config.featureToggles.lokiLogsDataplane = true;
```

Comment:
You may want to store and reset to the original state after the test.

See https://github.com/grafana/grafana/pull/96226/files#diff-9956354aca04b9ba46fbe2256d14bbf8567f20179b437be2074afd13fc19cf54R452